Cargo.toml

```diff
@@ -83,53 +83,53 @@ base64 = "0.13"
 encoding_rs = "0.8"
 futures-core = { version = "0.3.0", default-features = false }
 futures-util = { version = "0.3.0", default-features = false }
-http-body = "0.3.0"
-hyper = { version = "0.13.4", default-features = false, features = ["tcp"] }
+http-body = "0.4.0"
+hyper = { version = "0.14", default-features = false, features = ["tcp", "http1", "http2", "client"] }
```

Comment:
I'm undecided on this question: should reqwest allow optionally turning off http1 or http2? We don't have decide it now, we could make it optional in a future breaking change as part of a proposed general refactor, but curious what others think.

Reply:
If you think it's useful for hyper, extending the optionality into reqwest doesn't seem a big deal, and is potentially useful.

Reply:
To me, the USP of Reqwest (vs bare Hyper) is that things like this (just like, say, TLS) Just Work without having to think about it.

Reply:
This and the custom dns resolver support can be done in follow-up PRs. I'd like to keep this PR clean and self-contained.

Reply:
Yep, seems fair!